src/components/common/BottomButton.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import { FONT_STYLES } from '@src/styles/fontStyle';
+import { COLOR } from '@src/styles/color';
+
+type BottonButtonProps = {
+  width: number;
+  color: string;
+  text: string;
+};
```

Comment:
type BottomButtonProps = {
```suggestion
type BottonButtonProps = {
  width: number;
  color: string;
  text?: string;
};
```
로 하면 text를 optional 로 받을 수 있어요! styled-components에서도 같은 타입을 받을 수 있습니다! `StyledbuttonProps`로 지정해준 부분 코드를 줄일 수 있을 것 같아요 ㅎㅎ